src/sentry/mediators/sentry_apps/internal_creator.py

```diff
@@ -34,29 +34,30 @@ def call(self):
         self.sentry_app.verify_install = False
         self.sentry_app.save()
 
-        self._create_access_token()
         self._install()
+        self._create_access_token()
 
         return self.sentry_app
 
     def _create_access_token(self):
-        self.api_token = ApiToken.objects.create(
-            user=self.sentry_app.proxy_user,
-            application_id=self.sentry_app.application.id,
-            scope_list=self.sentry_app.scope_list,
-            expires_at=None,
```

Comment:
Let's make this an option to pass in and then default to `None` so we can use this for external integrations in the future